### DISCUSSION TITLE
Ventilation system using TESPy

### DISCUSSION THREAD

**Discussion by Nicolasgaete** [oemof/tespy#442] [Q&A] [answered]
I wanted to ask if anyone has simulated a ventilation system using TESPY. 

The system is a box with a "window" as the inlet for the air and 3 fans in the opposite end of the box. The box also has thermal resistances to simulate heat entering the box. I will attach a picture of the box.

<img width="682" alt="Screenshot 2023-10-12 at 16 08 55" src="https://github.com/oemof/tespy/assets/95881266/702bfc3d-34f1-41ab-9671-f46f6a5e38dc">

Do you think I can simulate this system using TESPy? I just found it and it will help me a lot.
Thank you in advance for your answers!

**Comment by fwitte** [comment]
Thanks for reaching out, what would you like to know about your system? TESPy is not exactly suited for super small scale systems but depending on what you want to know about it, it might work :).

Best

Francesco

**Reply by Nicolasgaete** [reply]
I need to know the Temperature over time, the idea is to vary the number of fans and the area of the air inlet to try to control the temperature

**Reply by fwitte** [reply, marked as answer]
In the sense that the box heats up over time due to heat input and affected by thermal inertia? TESPy is designed to give you outlet temperature as function of the heat input and air flow (or any other combinations of those), but only in steady state. Thermal inertia is not considered, which I think you are looking for, right?

**Reply by Nicolasgaete** [reply]
oh ok, yes, I am looking for the inertia... thanks anyways! best of luck
